wezterm-gui/src/overlay/copy.rs

```diff
@@ -161,9 +162,13 @@ impl CopyOverlay {
         render.dirty_results.add(search_row);
         render.update_search();
 
+        let shared_render: Arc<Mutex<CopyRenderable>> = Arc::new(Mutex::new(render));
+        let writer = SearchOverlayPatternWriter {render: Arc::clone(&shared_render)};
```

Comment:
I think the CI will flag this as not being formatted correctly.
You can use `make fmt` to automatically format (or, if you don't have `make`, you can more directly run: `cargo +nightly fmt`).
You will need nightly rustfmt to apply the preferred formatting.